src/buildCommon.js

```diff
@@ -692,16 +692,28 @@ const spacingFunctions: {[string]: {| size: string, className: string |}} = {
         size: "-0.16667em",
         className: "negativethinspace",
     },
+    "\\nobreak": {
+        size: "0em",
+        className: "nobreak",
+    },
+    "\\allowbreak": {
+        size: "0em",
+        className: "allowbreak",
+    },
 };
 
 // A lookup table to determine whether a spacing function/symbol should be
 // treated like a regular space character.
-const regularSpace: {[string]: boolean} = {
-    " ": true,
-    "\\ ": true,
-    "~": true,
-    "\\space": true,
-    "\\nobreakspace": true,
+const regularSpace: {[string]: { className?: string }} = {
```

Comment:
Can `{ className?: string }` be `{| className?: string |}`?

Reply:
This seemed to require `className` to be specified, i.e., flow then ignored the question mark. But perhaps a flow expert could comment...

Reply:
It seems to be this bug: https://github.com/facebook/flow/issues/2977

Reply:
I thought I saw it somewhere, and there it is 😄